test/import-export-nested-dir.spec.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+chai.use(require('dirty-chai'))
+const expect = chai.expect
+const IPLD = require('ipld')
+const pull = require('pull-stream/pull')
+const values = require('pull-stream/sources/values')
+const collect = require('pull-stream/sinks/collect')
+const map = require('async/map')
+const CID = require('cids')
+
+const importer = require('../src')
+const exporter = require('ipfs-unixfs-exporter')
+
+describe('import and export: directory', () => {
+  const rootHash = 'QmdCrquDwd7RfZ6GCZFEVADwe8uyyw1YmF9mtAB7etDgmK'
+  let ipld
+
+  before((done) => {
+    IPLD.inMemory((err, resolver) => {
+      expect(err).to.not.exist()
+
+      ipld = resolver
+
+      done()
+    })
+  })
+
+  it('imports', function (done) {
+    this.timeout(20 * 1000)
+
+    pull(
+      values([
+        { path: 'a/b/c/d/e', content: pull.values([Buffer.from('banana')]) },
```

Comment:
This is an exemplarily comment. There's some calls with `pull.*` in the tests. I don't have a clue why they work. I made some code changes on top of this PR and then the tests failed for me.

Anyway, I guess it makes sense to replace all occurrences of `pull.*` with there `*` counterparts.

Reply:
Interesting.  I'm going to merge this PR, will do another with the `pull.*` to `*` change, otherwise this runs the risk of becoming a 'general improvements' thread.

What were the changes you made that meant these tests stop working?

Reply:
I tried to find out what made things not working, but I couldn't. It is during my IPLD API changes stuff, so it's a lot of dependencies changing and also a lot of linking involved. Hence hard to tell what made it fail.